host_vars/stats-2025.yml

```diff
@@ -0,0 +1,356 @@
+certbot_auto_renew_minute: "{{ 59 |random(seed=inventory_hostname)  }}"
+certbot_auth_method: --webroot
+certbot_install_method: virtualenv
+certbot_virtualenv_package_name: python3-virtualenv
+certbot_auto_renew: yes
+certbot_auto_renew_user: root
+certbot_environment: production
+certbot_well_known_root: /srv/nginx/_well-known_root
+certbot_share_key_users:
+  - nginx
+  - influxdb
+certbot_post_renewal: |
+    systemctl restart nginx || true
+    systemctl restart influxdb || true
+certbot_domains:
+ - "{{ hostname }}"
+certbot_agree_tos: --agree-tos
+
+# NGINX
+nginx_enable_default_server: false
+nginx_servers:
+  #- influxdb
+  - redirect-ssl
+nginx_ssl_servers:
+  #- influxdb-ssl
+  - grafana-ssl
+nginx_conf_http:
+  client_max_body_size: 1g
+
+# Nginx Letsencrypt bindings
+nginx_ssl_role: usegalaxy_eu.certbot
+nginx_conf_ssl_certificate: /etc/ssl/certs/fullchain.pem
+nginx_conf_ssl_certificate_key: /etc/ssl/user/privkey-nginx.pem
+
+####### InfluxDB
+
+# General Influx
+influxdb_ver: 1.8.10 # stays the same as stats.usegalaxy.org.au - last available release of v1.x
+
+influxdb_base_data_dir: /mnt/influx_data
+influxdb_http_auth_enabled: "true"
+influxdb_http_https_enabled: "true"
+influxdb_http_https_certificate: "/etc/ssl/certs/fullchain.pem"
+influxdb_http_https_private_key: "/etc/ssl/user/privkey-influxdb.pem"
+influxdb_http_log_enabled: "false"
+
+# InfluxDB data
+influxdb_index_version: "tsi1"
+influxdb_data_cache_max_memory_size: 1048576000
+influxdb_data_cache_snapshot_write_cold_duration: "10s"
+influxdb_data_max_values_per_tag: 3000000
+
+# InfluxDB logging
+influxdb_logging_level: "error"
+
+# [[graphite]]
+influxdb_graphite_enabled: "false"
+influxdb_collectd_enabled: "false"
+influxdb_tsb_enabled: "false"
+influxdb_udp_enabled: "false"
+
+#Influx Users
+influxdb:
+  admin:
+    username: admin
+    password: "{{ vault_influx_admin_password }}"
+  node:
+    username: node
+    password: "{{ vault_influx_node_password }}"
+  grafana:
+    username: grafana
+    password: "{{ vault_influx_grafana_password }}"
+  url: "https://{{ hostname }}:8086"
+
+#Influx DB objects
+influxdb_databases:
+  # Clusters
+  - 'pulsar-mel2'
+  - 'pulsar-mel3'
+  - 'pulsar-paw'
+  - 'pulsar-nci-training'
+  - 'galaxy_etca'
+  - 'aarnet'
+  - 'pawsey'
+  - 'pulsar-QLD'
+  # Others
+  - 'pulsar-special'
+  - 'ancillary'
+  - 'GA_server'
+  - 'queues'
+  - 'staging'
+  - 'dev'
+  - 'mdu-phl'
+  - 'mdu-research'
+  - 'aarnet-testing'
+
+retention_policy_dbs:
+  # Clusters
+  - 'pulsar-mel2'
+  - 'pulsar-mel3'
+  - 'pulsar-paw'
+  - 'pulsar-nci-training'
+  - 'aarnet'
+  - 'pawsey'
+  - 'pulsar-QLD'
+  # Others
+  - 'pulsar-special'
+  - 'ancillary'
+  - 'queues'
+  - 'staging'
+  - 'dev'
+  - 'mdu-phl'
+  - 'mdu-research'
+
+retention_policy_8w_dbs:
+  - 'galaxy_etca'
+
+
+###### Grafana
+
+# TODO set version (installed version on Ubuntu 24.04 LTS) so doesn't update Grafana when role is executed again
```

Comment:
sudo apt upgrade might do that anyway. I think you might have done something somewhere to stop it from upgrading